terraform/modules/ingress/inputs.tf

```
@@ -0,0 +1,47 @@
+variable "container_name" {
+  type = string
+}
+
+variable "image" {
+  type = string
+}
+
+variable "tag" {
+  type = string
+}
+
+variable "network" {
+  type = string
+}
+
+variable "port" {
+  type = number
+}
+
+variable "tls" {
+  type = bool
+}
+
+variable "mtls" {
+  type = bool
+}
+
+variable "submitter_url" {
+  type = string
+}
+
+variable "submitter_port" {
+  type = number
+}
```

You could group those 2:

```suggestion
variable "submitter" {
  type = object({
    url  = string,
    port = number,
  })
}
```

I think you could do the same with some other parameters.